crates/aptos/src/move_tool/mod.rs

```diff
@@ -77,48 +78,51 @@ use transactional_tests_runner::TransactionalTestOpts;
 /// about this code.
 #[derive(Subcommand)]
 pub enum MoveTool {
+    Clean(CleanPackage),
     Compile(CompilePackage),
     CompileScript(CompileScript),
-    Init(InitPackage),
-    Publish(PublishPackage),
+    #[clap(subcommand)]
+    Coverage(coverage::CoveragePackage),
+    CreateResourceAccountAndPublishPackage(CreateResourceAccountAndPublishPackage),
+    Document(DocumentPackage),
     Download(DownloadPackage),
+    Init(InitPackage),
     List(ListPackage),
-    Clean(CleanPackage),
-    VerifyPackage(VerifyPackage),
+    Prove(ProvePackage),
+    Publish(PublishPackage),
     Run(RunFunction),
     RunScript(RunScript),
+    #[clap(subcommand, hide = true)]
+    Show(show::ShowTool),
```

Comment:
Please put refactors in separate commits

Reply:
Will do.